data/summary.go

```diff
@@ -0,0 +1,19 @@
+package data
+
+import (
+    "time"
+    "go.mongodb.org/mongo-driver/bson/primitive"
+)
+
+type Summary struct {
+    ID                 primitive.ObjectID  `json:"-" bson:"_id,omitempty"`
+	UserID             string              `json:"UserID" bson:"_userId"`
+
+	LastUpdated        time.Time   `json:"lastUpdated" bson:"lastUpdated"`
+	LastData           time.Time   `json:"lastData" bson:"lastData"`
+	LastUpload         time.Time   `json:"lastUpload" bson:"lastUpload"`
+	AverageGlucose     float64     `json:"avgGlucose" bson:"avgGlucose"`
+	TimeInRange        float64     `json:"timeInRange" bson:"timeInRange"`
```

Comment:
I think it would be good to include the thresholds for TIR in this, even if they are constants right now. That would allow clients to render the TIR and ranges:
![image](https://user-images.githubusercontent.com/1129108/127513341-f519c8b9-6b64-4375-b1ad-1a089daefed2.png)

More to the point though, we may want to allow people to customize their TIR thresholds. We already allow that in Tidepool Web:

![image](https://user-images.githubusercontent.com/1129108/127513580-a7917559-5bba-4f24-87f4-e71854dfd284.png)

Reply:
This is a larger discussion, though, I don't expect these ranges to be dynamic in the future due to the problem with them being variant, but storing them for the sake of clarity is a worthy cause.

Added them into the summary model as configurable constants for now.